runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/SimpleExecutionState.java

```diff
@@ -149,11 +149,40 @@ public String getLullMessage(Thread trackedThread, Duration millis) {
     return message.toString();
   }
 
+  @VisibleForTesting
+  public String getBundleLullMessage(Thread trackedThread, Duration millis) {
+    // TODO(ajamato): Share getLullMessage code with DataflowExecutionState.
```

Comment:
would be good to share within this class at least

Reply:
Following same pattern for lull getLullMessage. Let me know if you want to refactor both.